src/include/ipc4/base_fw.h

```diff
@@ -359,6 +359,12 @@ enum ipc4_fw_config_params {
 	IPC4_TELEMETRY_BUFFER_SIZE = 24,
 	/* HW version information  */
 	IPC4_BUS_HARDWARE_ID = 25,
+	/* Total memory size available in debug window */
+	IPC4_DEBUG_WINDOW_SIZE = 26,
+	/* Number of Realtime KPB output pins */
```

Comment:
Other entries in this enum have been explained with acronyms opened, so better to put "Number of realtime Key Phrase Buffer (KPB) output pins."

Reply:
thanks